WardenApi/src/main/java/com/salesforce/dva/warden/dto/Policy.java

```diff
@@ -1,5 +1,305 @@
 package com.salesforce.dva.warden.dto;
 
-public class Policy {
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonValue;
 
+import java.math.BigInteger;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Date;
+import java.util.List;
+
+/**
+ * Policy Dto.
+ *
+ * @author  Jigna Bhatt (jbhatt@salesforce.com)
+ * Failing to add comment.
+ */
+@SuppressWarnings( "serial" )
+@JsonIgnoreProperties( ignoreUnknown = true )
+public class Policy
+    extends Entity
+{
+    private String service;
+    private String name;
+    private List<String> owners = new ArrayList<String>(  );
+    private List<String> users = new ArrayList<String>(  );
+    private String subSystem;
+    private String metricName;
+    private TriggerType triggerType;
+    private Aggregator aggregator;
+    private List<Double> threshold;
+    private String timeUnit;
+    private Double defaultValue;
+    private String cronEntry;
+    private List<BigInteger> suspensionLevelIds = new ArrayList<BigInteger>(  );
+
+    public String getServcie(  )
+    {
+        return service;
+    }
+
+    public void setService( String service )
+    {
+        this.service = service;
+    }
+
+    public String getName(  )
+    {
+        return name;
+    }
+
+    public void setName( String name )
+    {
+        this.name = name;
+    }
+
+    public List<String> getOwner(  )
+    {
+        return owners;
+    }
+
+    public void setOwner( List<String> owner )
+    {
+        this.owners = owner;
+    }
+
+    public List<String> getUser(  )
```

Comment:
@jbhatt-salesforce The following two methods should be plural form.  getUsers() and setUsers().